sdk/canton/community/base/src/main/scala/com/digitalasset/canton/data/ActionDescription.scala

```diff
@@ -222,6 +222,7 @@ object ActionDescription extends HasProtocolVersionedCompanion[ActionDescription
             byKey,
             version,
             Option.when(protocolVersion >= ProtocolVersion.v6)(templateId),
+            interfaceId.filter(_ => protocolVersion >= ProtocolVersion.v6),
```

Comment:
`ProtocolVersion.v7`... probably applies to more places, won't bring it up anymore.